src/components/story-element.js

```diff
@@ -93,87 +93,99 @@ function StoryElementImage({
   story = {},
   imageWidths,
   imageDefaultWidth,
-  onClick = () => {}
+  onClick = () => {},
 }) {
-  return React.createElement(
-    "figure",
-    {},
-    React.createElement(ResponsiveImage, {
-      slug: element["image-s3-key"],
-      metadata: element["metadata"],
+  let foo = '';
+
+  if (typeof window === 'undefined' || !element || !story) {
+    foo = React.createElement('img', {
+      src:
+        'https://images.prothomalo.qtstage.io/prothomalo-english%2F2020-10%2Fa2e5f7c5-76c3-4dd7-8558-4883ce57d3a0%2FScreenshot_2020_10_13_at_11_06_21_AM.png?w=500',
```

Comment:
why is the palo link there ? this can be passed as a config from the app level

Reply:
and have a default image here

Reply:
The ticket is saying `image container should have default height and width` it's not saying to add default image here. also if you will use a .png image it might be lead to some performance issue. 

You can add the default image to your app-level itself. I have commented on the ticket already check it.